packages/hooks/src/stories/UseTransaction.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, Text } from '@chakra-ui/react';
+import React from 'react';
+import { NETWORKS, Provider, useWallet } from '..';
+import { useTransaction } from '../hooks';
+
+export default {
+  title: 'Hooks/useTransaction',
+};
+
+export const Default = () => {
+  const giveAlert = (name, greeting) => {
+    alert('Hello ' + name + '! ' + greeting);
+  };
```

Comment:
This hook is meant to be for Ethereum contract calls. Can you add an example that actually calls an Ethereum contract? You can use `useContract`

Reply:
You can remove these current stories

Reply:
> Sure, I'll use `useContract` and remove current stories